sdks/python/apache_beam/runners/interactive/utils.py

```diff
@@ -451,21 +450,29 @@ def assert_bucket_exists(bucket_name):
     Logs a warning if the bucket cannot be verified to exist.
   """
   try:
-    from apitools.base.py.exceptions import HttpError
-    storage_client = storage.StorageV1(
-        credentials=auth.get_service_credentials(PipelineOptions()),
-        get_credentials=False,
-        http=get_new_http(),
-        response_encoding='utf8')
-    request = storage.StorageBucketsGetRequest(bucket=bucket_name)
-    storage_client.buckets.Get(request)
-  except HttpError as e:
-    if e.status_code == 404:
+    from google.cloud.exceptions import ClientError
+    from google.cloud.exceptions import NotFound
+    from google.cloud import storage
+    credentials = auth.get_service_credentials(PipelineOptions())
+    if credentials:
+      storage_client = storage.Client(
+          credentials=credentials.get_google_auth_credentials(),
+          project=None,
```

Comment:
see my above comment about the project option

Reply:
Done.